From 61a613d3b30d6d35ed12793deb8db0a8e59acd81 Mon Sep 17 00:00:00 2001
From: Tapasweni Pathak <tapaswenipathak@gmail.com>
Date: Thu, 7 Mar 2019 00:49:54 +0530
Subject: [PATCH] drivers: scsi: Fix dead store

off having value of vscnprintf is not being used after the statement so
can be dropped.
---
 drivers/scsi/scsi_logging.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/scsi/scsi_logging.c b/drivers/scsi/scsi_logging.c
index bd70339..eecd70f 100644
--- a/drivers/scsi/scsi_logging.c
+++ b/drivers/scsi/scsi_logging.c
@@ -112,7 +112,6 @@ void sdev_prefix_printk(const char *level, const struct scsi_device *sdev,
 				 "[%s] ", name);
 	if (!WARN_ON(off >= logbuf_len)) {
 		va_start(args, fmt);
-		off += vscnprintf(logbuf + off, logbuf_len - off, fmt, args);
 		va_end(args);
 	}
 	dev_printk(level, &sdev->sdev_gendev, "%s", logbuf);
-- 
2.10.0

